### PR TITLE
Turn off performanceObserver API

### PR DESCRIPTION
**Pull request by zhouyx**
temp solution to #9749

### CODE REVIEW

File: src/service/jank-meter.js [hunk 1]
```diff
@@ -182,7 +182,10 @@ function isJankMeterEnabled(win) {
  * @returns {boolean}
  */
 function isLongTaskApiSupported(win) {
-  return !!win.PerformanceObserver
+  // TODO (@lannka, #9749) Turn performanceObserver on.
+  const tempOff = false;
```
Comment (camelburrito) [inline src/service/jank-meter.js]: just return false?

Comment (zhouyx) [inline src/service/jank-meter.js]: I am trying to keep the old so it would convenient to turn it on later. But sure, I am good with `returning false`

Comment (jridgewell) [inline src/service/jank-meter.js]: Could just use two return statements?

```js
// TODO
return false;
return ...

**Comment by zhouyx** [comment]
It's batterApi not performanceObserver API that's throwing error. close by #9780

File: src/service/jank-meter.js (hunk 1 above)
Comment (zhouyx) [inline src/service/jank-meter.js]: just find: type check will complain about unreachable code.

**Pull request closed without merging**